src/Tribe/Service_Providers/Context.php

```diff
@@ -270,6 +270,38 @@ static function ( $post_type ) {
 					Tribe__Context::REQUEST_VAR => [ TEC::TAXONOMY ],
 				],
 			],
+			'view_url' => [
+				'read'  => [
+					Tribe__Context::FUNC => [
+						static function () {
+							return Utils\View::get_data( 'url', Tribe__Context::NOT_FOUND );
+						},
+					],
+					Tribe__Context::LOCATION_FUNC => [
+						// Handles the datepicker request.
+						'view_data',
+						static function ( $data ) {
+							$date_k = 'tribe-bar-date';
+							return is_array( $data ) && isset($data[ $date_k ])
+								? add_query_arg( [ $date_k => $data[ $date_k ]], tribe_get_request_var( 'url', home_url() ) )
+								: Tribe__Context::NOT_FOUND;
+						}
+					],
+				],
+			],
+			'view_prev_url' => [
+				'read'  => [
```

Comment:
:warning: Array double arrow not aligned correctly; expected 1 space(s) between "'read'" and double arrow, but found 2.

<sub>hash: 39d3fc70bac293d46c90f9b6ededbe69</sub>